nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -81,6 +81,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - [systemd-lock-handler](https://git.sr.ht/~whynothugo/systemd-lock-handler/), a bridge between logind D-Bus events and systemd targets. Available as [services.systemd-lock-handler.enable](#opt-services.systemd-lock-handler.enable).
 
+- [Mealie](https://nightly.mealie.io/), a self-hosted recipe manager and meal planner with a RestAPI backend and a reactive frontend application built in NuxtJS for a pleasant user experience for the whole family. Available as [services.mealie](#opt-services.mealie.enable)
```

Comment:
[REST](https://en.wikipedia.org/wiki/REST) is a fixed term. So you can say "REST API" instead of "RestAPI backend".

Reply:
Copy-pasted from the website, so I should maybe tell this to the creator as well :sweat_smile: